src/moduler/aktivitet/visning/avtalt-container/avtalt-form.tsx

```diff
@@ -16,7 +16,7 @@ export const SEND_FORHANDSORIENTERING = 'send_forhandsorientering';
 export const SEND_PARAGRAF_11_9 = 'send_paragraf_11_9';
 export const IKKE_SEND_FORHANDSORIENTERING = 'ikke_send_forhandsorientering';
 
-function validateForhandsorienter(val, values): string | undefined {
+function validateForhandsorienter(val: string, values: { avtaltSelect?: string }): string | undefined {
```

Comment:
Kan du ikke normalisere typene? Altså at det er en string som returneres og da er det tom streng som gjelder for manglende verdi. Syns det er vanskeligere å forholde seg til typer når man har mange kombinasjoner av returtyper

Reply:
Hvis du returnerer tom string her kommer feilmeldingen opp

Reply:
samme gjelder alle validate funksjonene